code/game/turfs/simulated/floor.dm

```diff
@@ -30,10 +30,14 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 	var/broken = 0
 	var/burnt = 0
 	var/floor_tile = null //tile that this floor drops
-	var/list/broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5")
-	var/list/burnt_states = list()
+	var/list/broken_states
+	var/list/burnt_states
 
 /turf/open/floor/New()
+	if (!broken_states)
+		broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5")
```

Comment:
This is not related to the PR, but these states are mostly the same for most turfs in the game, wouldn't it make more sense to pull this information from another place if the turf doesn't have it defined?